packages/contracts/docker/setup.sh

```
@@ -103,10 +104,29 @@ wait_for_river_chain() {
 
 deploy_contracts() {
   pushd ./core
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'pushd' is undefined.

(SC3044)